plugins/org.wso2.developerstudio.visualdatamapper.diagram/src/org/wso2/developerstudio/datamapper/diagram/part/DataMapperDiagramEditor.java

```diff
@@ -496,6 +502,38 @@ private boolean updateAssociatedConfigFile(IProgressMonitor monitor) {
 		}
 		return true;
 	}
+	
+	private void updateAssociatedXsltFile(IProgressMonitor monitor) {
+		IEditorInput editorInput = this.getEditorInput();
+        if (editorInput instanceof IFileEditorInput) {
+            IFile diagramFile = ((FileEditorInput) editorInput).getFile();
+            String diagramFilePath = diagramFile.getFullPath().toString();
+            String schemaFilePath = diagramFilePath.replaceAll(".datamapper_diagram$", ".datamapper"); //$NON-NLS-1$ //$NON-NLS-2$
+            String xsltStyleSheetFilePath = diagramFilePath.replaceAll(".datamapper_diagram$", "_xsltStyleSheet.xml");
+            IFile schemaFile = diagramFile.getWorkspace().getRoot().getFile(new Path(schemaFilePath));
+            IFile xsltStyleSheetFile = diagramFile.getWorkspace().getRoot().getFile(new Path(xsltStyleSheetFilePath));
+            InputStream schemaInPutStream = null;
+            if (xsltStyleSheetFile.exists() && schemaFile.exists()) {
+                try {
```

Comment:
We can use try with resource and remove the finally block. https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html